src/tests/ftest/container/multiple_delete.py

```diff
@@ -30,7 +34,8 @@ def test_multiple_container_delete(self):
 
         :avocado: tags=all,full_regression
         :avocado: tags=hw,large
-        :avocado: tags=container,multi_container_delete
+        :avocado: tags=container
+        :avocado: tags=multi_container_delete
```

Comment:
We should also add the test method tag:
```suggestion
        :avocado: tags=multi_container_delete,test_multiple_container_delete
```

Reply:
Done